README.md

```diff
@@ -74,43 +74,31 @@ Examples of `Build` resource using the example strategies installed by default.
 ## Try it!
 
 * Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster.
-* Clone this repository from GitHub at the v0.3.0 tag:
+* Install [Tekton](https://cloud.google.com/tekton) `v0.20.1`:
+    ```bash
+    $ kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.20.1/release.yaml
+    ```
 
-  ```bash
-  $ git clone --branch v0.3.0 https://github.com/shipwright-io/build.git
-  ...
-  $ cd build/
-  ```
-
-  _Coming soon - install Shipwright Build via kubectl!_
-
-* Install [Tekton](https://cloud.google.com/tekton) by running [hack/install-tekton.sh](hack/install-tekton.sh), it installs v0.20.1.
+* Install Shipwright build controller `v0.3.0`:
 
   ```bash
-  $ hack/install-tekton.sh
+  $ kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.3.0/release.yaml
   ```
 
-* Install Shipwright and sample strategies via `make`:
+* Install Buildpacks sample build strategy:
 
   ```bash
-  $ make install
-  ```
-
-* Add a push secret to your container image repository, such as one on Docker Hub or quay.io:
-
-  ```yaml
-  $ kubectl create secret generic push-secret \
-  --from-file=.dockerconfigjson=$HOME/.docker/config.json \
-  --type=kubernetes.io/dockerconfigjson
+  $ kubectl apply --filename https://raw.githubusercontent.com/shipwright-io/build/v0.3.0/samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml
```

Comment:
I had run into this recently too ... @zhangtbj got around to fixing it before I did :-)

Reply:
Thanks, Gabe! :)